Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -62,39 +62,40 @@ namespace EMotionFX
                 {
                     editContext->Class<ActorComponent::BoundingBoxConfiguration>("Actor Bounding Box Config", "")
                         ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
-
                         ->DataElement(AZ::Edit::UIHandlers::ComboBox, &ActorComponent::BoundingBoxConfiguration::m_boundsType,
                                       "Bounds type",
-                                      "The method used to compute the Actor bounding box. NOTE: ordered by least expensive to compute to most expensive to compute."
-                        )
-                        ->EnumAttribute(ActorInstance::BOUNDS_STATIC_BASED, "Static bounds (source-asset bounds)")
-                        ->EnumAttribute(ActorInstance::BOUNDS_NODE_BASED, "Bone position-based")
-                        ->EnumAttribute(ActorInstance::BOUNDS_NODEOBB_BASED, "Bone local bounding box-based")
-                        ->EnumAttribute(ActorInstance::BOUNDS_MESH_BASED, "Render mesh vertex position-based (VERY EXPENSIVE)")
-
-                        ->DataElement(0, &ActorComponent::BoundingBoxConfiguration::m_autoUpdateBounds,
+                                      "The method used to compute the Actor bounding box. NOTE: ordered by least expensive to compute to most expensive to compute.")
+                            ->EnumAttribute(ActorInstance::BOUNDS_STATIC_BASED, "Static (Recommended)")
+                            ->EnumAttribute(ActorInstance::BOUNDS_NODE_BASED, "Bone position-based")
+                            ->EnumAttribute(ActorInstance::BOUNDS_MESH_BASED, "Mesh vertex-based (Expensive)")
+                            ->Attribute(AZ::Edit::Attributes::ChangeNotify, AZ::Edit::PropertyRefreshLevels::EntireTree)
+                        ->DataElement(AZ::Edit::UIHandlers::Default, &ActorComponent::BoundingBoxConfiguration::m_expandBy,
+                            "Expand by",
+                            "Percentage that the calculated bounding box should be automatically expanded with. "
+                            "This can be used to add a tolerance area to the calculated bounding box to avoid clipping the character too early. "
+                            "A static bounding box together with the expansion is the recommended way for maximum performance. (Default = 25%)")
+                            ->Attribute(AZ::Edit::Attributes::Suffix, " %")
+                            ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
```

Comment:
Expanding by a negative percentage should effectively contract the bbox. It's important that if a parameter has a range of sensible values, it may take all values in that range, even if the use case for some values isn't clear. The math in ActorInstance will work for [-1, inf), although -1 is totally useless as it causes the actor to never render, so a minimum of e.g. -0.99 is more realistic.

Maybe a soft-min of 0 instead to cover the "expected" use case.

Reply:
Good point. I changed it to -100+tolerance ~ -99.999% as the UI values are in % and get converted to normalized values internally. This basically allows it to shrink to a super small aabb but not scale it negatively.